portal-web/test/functional/com/liferay/portalweb/tests/enduser/publications/PublicationsAssetList.testcase

```diff
@@ -134,4 +134,57 @@ definition {
 		}
 	}
 
+	@description = "Assert that is possible to edit a Asset List within Publications"
+	@priority = 3
+	test CanEditAssetList {
+		task ("Create a manual asset list") {
+			JSONAssetlist.addManualAssetListEntry(
+				groupName = "Guest",
+				title = "Manual Asset List Test");
+		}
+
+		task ("Add new Publication and use it") {
+			JSONPublications.addPublication(publicationName = "Publication Name");
+
+			JSONPublications.selectPublication(publicationName = "Publication Name");
+		}
+
+		task ("Rename the asset list") {
+			AssetListsAdmin.openAssetListsAdmin(siteURLKey = "guest");
+
+			AssetListsAdmin.renameAssetList(
+				newAssetListTitle = "Manual Asset List Edit",
+				oldAssetListTitle = "Manual Asset List Test");
+		}
+
+		task ("Assert that it's possible to publish it") {
+			PublicationsNavigator.gotoReviewChanges();
+
```

Comment:
Can you add the steps to review the changes under Asset List Entry, you can toggle Show All Items to make them show up.